app/src/main/res/layout/activity_main.xml

```diff
@@ -51,6 +51,7 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:fillViewport="true"
+            android:layout_marginBottom="60dp"
```

Comment:
@HaripriyaB You can move this hardcoded value to 'dimens '

Reply:
sure will do it right now.